### PR TITLE
Package.json Script Purge

### PR DESCRIPTION
**Pull request by gdjohnson**
This branch purges unused script aliases in the `commonwealth/package.json` file. It also removes the relevant documentation.

For context on decisions made, see the discussion thread on #4970. If we want to take a more aggressive approach, and prune further, there are some on-the-fence scripts that were retained due to engineer interest (e.g. some of the Sequelize migration scripts) which could be removed. 

Cleaning up linter scripts has expanded in scope—there's no clear team consensus on which scripts and libraries are used, and we should specifically figure out a linting strategy and remove unwanted packages/aliases then. This will further reduce our count.

Closes: #4893.

### CODE REVIEW

**Review by jnaviask — APPROVED** [review]
Off to a great start. Some additional comments to work with:

- Flagging dump-db-limit / load-db-limit (and associated .sql files) for removal, as the NotificationsRead table is sufficiently small for us to simply do a dump of the entire db. We may want to replace it with a yarn dump-all command to get a full dump.
- How is load-db-local diff than load-db? Do we need a load-db-local? Hell, when do we use dump-db-local?
- We may want to consider removing the docker scripts -- not sure how/when we will use them, especially post chain events deprecation. cc @timolegros
-

**Comment by gdjohnson** [comment]
We're down to 55 scripts... Once linter stuff is audited we'll approach high-40s... Can we get lower... I'm picturing a game of limbo...

**Comment by timolegros** [comment]
> Off to a great start. Some additional comments to work with:
> 
> * Flagging dump-db-limit / load-db-limit (and associated .sql files) for removal, as the NotificationsRead table is sufficiently small for us to simply do a dump of the entire db. We may want to replace it with a yarn dump-all command to get a full dump.
> * How is load-db-local diff than load-db? Do we need a load-db-local? Hell, when do we use dump-db-local?
> * We may want to consider removing the docker scripts -- not sure how/when we will use them, especially post chain events deprecation. cc @timolegros

- Docker scripts in `commonwealth/package.json` can be removed. We've moved on in favour of running docker scripts locally rather than remotely from root `package.json` via `start-rmq` and `start-redis`.
- Local db scripts were especially useful during the creation of chain-events and removal of chain-event types. These can/should be removed.
- I'm not so sure about removing the partial dumps. The NotificationsRead table may be small now but it could grow significantly. Rather than removing all the different dump types we should consider having a single `dump-db` script which takes an optional argument that is, for example, one of 'all', 'limit', or 'semi' with a default to semi which is what the current `dump-db` script does. Would still allow dumping the entire db via 'all' or getting partial tables via the 'limit' option. The script can take inspiration from the existing `load-db` script which allows loading dumps of different names but defaults to `latest.dump`.

@jnaviask @gdjohnson

**Comment by jnaviask** [comment]
@gdjohnson are we good to merge this branch, or do we anticipate more changes on it?

**Comment by gdjohnson** [comment]
@jnaviask Merge is really pending a decision on the above, hence why I tagged ya last week!

The uncontroversial decisions have been acted upon (e.g. removing local db & docker scripts). However, @timolegros's partial dump DB recommendations have not been acted upon, in part because they clashed with your earlier recommendation, and in part because Forest expressed some ambivalence, in stand-up last week, whether writing db scripts was within my purview as documentarian.

I'm happy with anything—just caught in a crossfire of different opinions / recommendations, all from sources either better-qualified or higher-ranking than me to make the call. 

It's a bit silly, because this is such a small-peas decision in the grand scheme of things, and we've probably used as much time talking back-and-forth at this point as it would've taken for me to implement Tim's rec. But here we are! Maybe some meta-level takeaways.

**Comment by timolegros** [comment]
> @jnaviask At this point, I think we should:
> 
> * Merge
> * Open a ticket for Tim's partial-db recommendations
> 
> Feel free to veto or counter-propose. Just tryna follow the "Close your own ticket" philosophy in our Ways of Working doc.

@gdjohnson I suggest not removing the `limited_dump.sql` script in this PR since it can be extremely useful as the size of the database grows. Paired with #5245 which I just created, we should be able to get a clean `dump-db` interface and `package.json` while maintaining accessibility to useful functions/versions of db dumps.

**Comment by gdjohnson** [comment]
Thanks, done!

File: packages/commonwealth/package.json [hunk 1]
```diff
@@ -49,23 +39,16 @@
...
-    "test-client": "webpack-dev-server --config webpack/webpack.config.test.js",
-    "test-consumer": "ts-mocha --project tsconfig.json test/systemTests/consumer.test.ts --timeout 20000",
     "test-devnet": "nyc ts-mocha --project tsconfig.json './test/devnet/**/*.spec.ts'",
     "test-e2e": "TEST_ENV=playwright npx playwright test -c ./test/e2e/playwright.config.ts --workers 2 ./test/e2e/e2eRegular/*",
-    "test-e2e-serial": "TEST_ENV=playwright npx playwright test --workers 1 ./test/e2e/e2eSerial/*",
     "test-e2e-mature": "TEST_ENV=playwright npx playwright test -c ./test/e2e/playwright.config.ts --workers 2 ./test/e2e/mature/*",
-    "test-events": "nyc ts-mocha --project tsconfig.json ./test/integration/events/*.spec.ts",
+    "test-e2e-serial": "TEST_ENV=playwright npx playwright test --workers 1 ./test/e2e/e2eSerial/*",
     "test-integration-util": "NODE_ENV=test nyc ts-mocha --project tsconfig.json ./test/integration/*.spec.ts",
     "test-query": "ts-node server/scripts/testQuery.ts",
```
Comment (timolegros) [inline packages/commonwealth/package.json:53]: `test-query` yarn command + `testQuery.ts` script should be removed.

Comment (timolegros) [inline packages/commonwealth/package.json:53]: `test-query` yarn command + `testQuery.ts` script should be removed.

**Review by timolegros — CHANGES_REQUESTED** [review]
LGTM just requesting one little additional deletion for a script that should have never made it into the codebase to begin with

**Comment by gdjohnson** [comment]
@timolegros Amazing, thanks Tim for the eyeballs. 

Updated!

**Review by timolegros — APPROVED** [review]

**Comment by gdjohnson** [comment]
Merging, since 2 approvals!

**Pull request merged**